src/documentation/pages/components/modal/examples/_alert.vue

```diff
@@ -0,0 +1,95 @@
+<template lang="pug">
+  <ComponentExample title="Alert" id="alert" padding="-p--0" :tabs="exampleTabs">
+    .chi-backdrop.-p--6.-position--relative.-z--0(slot="example")
+      .chi-backdrop__wrapper
+        section.chi-modal.-portal(role="dialog", aria-label="Modal description", aria-modal="true")
+          header.chi-modal__header
+            h2.chi-modal__title Alert Modal Title
+            button(class='chi-button -icon -close', aria-label='Close')
+              .chi-button__content
+                i.chi-icon.icon-x(aria-hidden="true")
+          .chi-modal__content
+            .-d--flex
+              i.chi-icon.icon-circle-x.-sm--3.-icon--danger.-mr--2(aria-hidden="true")
+              .-w--100
+                p.-text--bold.-m--0 Replace this subhead text with specific text from project documentation
+                p.-mt--1.-mb--0
+                  | Supporting text to explain the subhead text. Check user story to customize based on requirements.
+                  | This is placeholder text to show spacing and line height. Replace this text with text provided in requirements.
+          footer.chi-modal__footer
+            button.chi-button.-primary Button
+    <pre class="language-html" slot="code-webcomponent">
+      <code v-highlight="$data.codeSnippets.webcomponent" class="html"></code>
+    </pre>
+    <Wrapper slot="code-htmlblueprint">
+      <JSNeeded />
+      <pre class="language-html">
+        <code v-highlight="$data.codeSnippets.htmlblueprint" class="html"></code>
+      </pre>
+    </Wrapper>
+  </ComponentExample>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator';
+
+declare const chi: any;
+
+@Component({
+  data: () => {
+    return {
+      exampleTabs: [
+        {
+          disabled: true,
+          id: 'webcomponent',
+          label: 'Web component',
+        },
+        {
+          active: true,
```

Comment:
no need to use active: true here since the above one is disabled hence this tab will resort to active by default

Reply:
this is actually OK sorry, don't change it